model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -251,34 +251,45 @@ case class WorkspaceStatus(workspaceName: WorkspaceName, statuses: Map[String, S
 
 case class BucketUsageResponse(usageInBytes: BigInt)
 
-case class ErrorReport(source: String, message: String, statusCode: Option[StatusCode], causes: Seq[ErrorReport], stackTrace: Seq[StackTraceElement], exceptionClass: Option[Class[_]])
+case class CErrorReport(source: String, message: String, statusCode: Option[StatusCode], causes: Seq[CErrorReport], stackTrace: Seq[StackTraceElement], exceptionClass: Option[Class[_]])
```

Comment:
I also don't like CErrorReport. But I strongly dislike RawlsErrorReport: we
are in the Rawls codebase, so of course it's a Rawls error report. It's
unnecessarily wordy and redundant.

The philosophically ideal answer IMO is to call them both ErrorReport and
figure out the imports. But I think that makes it a pain for devs and leads
to weird syntax things to disambiguate, though I confess I haven't checked.